src/components/calcite-popover/calcite-popover.tsx

```diff
@@ -0,0 +1,160 @@
+import { Component, Element, Host, Method, Prop, State, Watch, h } from "@stencil/core";
+
+import { CalcitePlacement } from "../interfaces";
+
+// https://www.npmjs.com/package/popper.js
+// https://popper.js.org/
+import Popper from "popper.js";
+
+@Component({
+  tag: "calcite-popover",
+  styleUrl: "calcite-popover.scss",
+  shadow: true
+})
+export class CalcitePopover {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * Determines where the element will be positioned.
+   * horizontal: Positioned to the left or right of the positionElement.
+   * vertical: Positioned above or below the positionElement.
+   */
+  @Prop({ reflect: true }) placement: CalcitePlacement = "horizontal";
```

Comment:
Going with horizontal/vertical because it will position it left/right depending on space. 

It sounds better than `above or below` or `left or right`